docs/source/analyze/node-resources.md

```diff
@@ -124,3 +125,25 @@ spec:
           - pass:
               message: This cluster has a node with enough memory.
 ```
+
+```yaml
+    - nodeResources:
+        checkName: Must have at least 3 nodes with 2 cores and labels kubernetes.io/hostname=docker-desktop and kubernetes.io/role=database-primary-replica
```

Comment:
I wouldn't suggest putting the labels in the checkName; these should be human-readable. Maybe:

```suggestion
        checkName: Must have at least 3 nodes with 2 cores, with Docker Desktop and a database
```

Reply:
I don't understand how this check actually works. Where's the check for the labels? Does it filter the nodes by ones with those labels such that count() < 3 means there must be 3 nodes with docker-desktop on them?

Reply:
The main Idea is to be sure that the nodes being analyzed have certain labels. Maybe the example I used was miss-leading, check the new one in the new commit. In this case, for a node to match the filter this conditions must be met:
- Have 2 cores
- Have a label "kubernetes.io/hostname"="docker-desktop"
- Have a label "kubernetes.io/role"="database-primary-replica"

Reply:
OK, so it sounds like all filters get applied, so it means that we need 3 nodes that have those labels and minimum cores, etc. That makes sense, but frankly is a bit confusing. If you're going to go that route, you need to change the actual messages. Currently, someone could have 3 nodes, but you'll tell them they don't, just because they don't have 3 nodes **with** those labels. This might be better as a different example, or even separate examples.

Reply:
I've written a new, simpler example, only using the selector matchLabel (see last commit). Let me know what you think